source/chapters/setup.rst

```diff
@@ -66,13 +73,58 @@ out**.
       microphone is connected to
 #. Click :guilabel:`Apply` to save the changes.
 
-This minimal configuration may be suitable for **radio DJs** that do not need
-headphone :term:`cueing`.
+This minimal configuration may be suitable for **radio DJs** who do not need
+headphone cueing.
+
+.. _setup-laptop-with-splitter:
+
+Laptop and Splitter Cable
+=========================
+
+.. figure:: ../_static/mixxx_setup_splitter_adaptors.png
+   :align: center
+   :width: 75%
+   :figwidth: 100%
+   :alt: Using Mixxx with your built-in sound card and a DJ splitter cable
+   :figclass: pretty-figures
+
+
+The cheapest way to DJ and :term:`cue` with headphones uses a 
+stereo-to-mono DJ splitter cable (also known as a “Y cable”) plugged
+into the headphone jack of a laptop or other computer. This cable divides the
+stereo output from a single jack into two separate mono singals.
+
+This setup allows you to start DJing without having to invest in expensive 
+equipment. However, your audience will not be able to enjoy music producers' 
+artistic use of stereophonic sound (although many club PA systems are wired in 
+mono anyway). Furthermore, soundcards built into computers are usually low
+quality.
+
+**Using a built-in soundcard and a stereo-to-mono splitter**
+
+#. Open :menuselection:`Preferences --> Sound Hardware`
+#. Select the :guilabel:`Output` tab
+#. Check that your headphones are plugged into the side of the cable with a
+   headphones symbol and that the speakers are plugged into the side of the cable
+   with a speaker symbol.
+#. From the :guilabel:`Master` drop-down menu, select your built-in soundcard,
+   then :guilabel:`Channel 1`
+#. From the :guilabel:`Headphones` drop-down menu, select your built-in 
+   soundcard, then :guilabel:`Channel 2`
+#. Click :guilabel:`Apply` to save the changes.
+
+.. warning:: Most splitter cables, particularly those marketed as headphone
```

Comment:
AFAIK, there are only two commercially available DJ splitter cables. See http://mixxx.org/wiki/doku.php/hardware_compatibility#splitter_cables

Come to think of it, I'll add that link to the manual.